### DISCUSSION TITLE
Does training the model is not necessary?

### DISCUSSION THREAD

**Discussion by naknak-choi** [AI21Labs/Parallel-Context-Windows#7] [Q&A]
Hi there!

I've been reviewing the code and noticed that I couldn't find the training code. Does this mean that when using the PCW method, training the model is not necessary? Is it enough to just prepend the context window to the attention mechanism?

**Comment by nirrai21** [comment]
Hi!
That's exactly right, one of the advantages of PCW is that no training is needed at all. Please feel free to read the [paper](https://arxiv.org/abs/2212.10947) for further information.

**Reply by naknak-choi** [reply]
Thank you!!
